dependencies.yaml

```diff
@@ -190,48 +213,78 @@ dependencies:
           - matrix:
             packages:
               - python>=3.9,<3.11
-  run_python:
+  run_python_ucxx:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
-          - numpy>=1.21
+          - &numpy numpy>=1.21
           - pynvml>=11.4.1
       - output_types: [conda]
         packages:
           - ucx
+  run_python_distributed_ucxx:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - &numba numba>=0.57.1
+          - rapids-dask-dependency==24.4.*
   test_cpp:
     common:
       - output_types: conda
         packages:
           - *cmake_ver
           - *gtest
           - *gmock
-  test_python:
+  test_python_ucxx:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cloudpickle
-          - dask
-          - distributed
-          - numba>=0.57.1
+          - *numba
           - pytest
           - pytest-asyncio
           - pytest-rerunfailures
```

Comment:
```suggestion
          - pytest-rerunfailures
          - rapids-dask-dependency==24.4.*
```

Reply:
I think this is not needed - see above. https://github.com/rapidsai/ucxx/pull/174#discussion_r1466735058